apps/console/src/features/groups/components/edit-group/edit-group-roles.tsx

```diff
@@ -16,972 +16,136 @@
-                                                        isItemChecked={ checkedAssignedListItems.includes(role) }
-                                                        showSecondaryActions={ false }
-                                                        data-testid="group-mgt-update-roles-modal-selected-roles"
-                                                    />
-                                                );
-                                            }
-                                        })
-                                    }
-                                </TransferList>
-                            </TransferComponent>
-                        </Modal.Content>
-                    )
-            }
-            <Modal.Actions>
-                <Grid>
-                    <Grid.Row column={ 2 }>
-                        <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 8 }>
-                            <LinkButton
-                                data-testid="group-mgt-update-roles-modal-cancel-button"
-                                floated="left"
-                                onClick={ handleCloseAddNewGroupModal }
-                            >
-                                { t("common:cancel") }
-                            </LinkButton>
-                        </Grid.Column>
-                        <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 8 }>
-                            <PrimaryButton
-                                data-testid="group-mgt-update-roles-modal-save-button"
-                                floated="right"
-                                onClick={ () => updateUserRole(group, tempRoleList) }
-                                loading={ isSubmitting }
-                                disabled={ isSubmitting }
-                            >
-                                { t("common:save") }
-                            </PrimaryButton>
-                        </Grid.Column>
-                    </Grid.Row>
-                </Grid>
-            </Modal.Actions>
-        </Modal>
-    );
-
-    const handleAssignedRoleListSearch = (e: React.ChangeEvent<HTMLInputElement>, { value }: InputOnChangeData) => {
-        let isMatch: boolean = false;
-        const filteredRoleList:RolesMemberInterface[] = [];
-
-        if (!isEmpty(value)) {
-            const re: RegExp = new RegExp(escapeRegExp(value), "i");
-
-            assignedRoles && assignedRoles?.map((role: RolesMemberInterface) => {
-                const groupName: string[] = role.display.split("/");
-
-                if (groupName.length > 1) {
-                    isMatch = re.test(role.display);
-                    if (isMatch) {
-                        filteredRoleList.push(role);
-                        setAssignedRoles(filteredRoleList);
+    const getPlaceholders = () => {
+        if (showEmptyRolesListPlaceholder) {
+            return (
+                // TODO: Need to replace the i18N with the correct one.
```

Comment:
Shall we resolve the ToDo comments?

Reply:
This is fixed, will remove the comment.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4326/commits/317543c8be8c0461214e6e67425f492c835d6bd7